enterprise/frontend/src/embedding-sdk/test/storybook-themes.ts

```diff
@@ -7,16 +7,14 @@ export const darkColors = {
   lighterGrey: "#E3E7E4",
   lightGrey: "#ADABA9",
   darkGrey: "#3B3F3F",
-  background: "#151C20",
+  background: "#161A1D",
 };
 
 export const darkTheme = defineMetabaseTheme({
   fontFamily: "Lato",
   fontSize: "14px",
   colors: {
     brand: darkColors.primary,
-    "brand-hover": darkColors.darkGrey,
-    "brand-hover-light": darkColors.darkGrey,
```

Comment:
Let them be auto-generated by dynamic CSS variables. We can probably do auto-generation for many of these to be honest, but let's save that for another PR.